src/main.js

```diff
@@ -22,16 +22,40 @@ saveButton.addEventListener("click", createNewIdeaCard);
 // titleInput.addEventListener("keydown", enableSaveButton);
 bodyInput.addEventListener("keydown", enableSaveButton);
 ideaCardContainer.addEventListener("click", determineIdeaCardEvent);
-window.addEventListener("load", getStoredIdeas);
+window.addEventListener("load", onload);
 
 // FUNCTIONS:
 
+function onload() {
+  getStoredIdeas();
+  displayUserCards(userIdeas);
+}
+
 function showStarredIdeas() {
   toggleButtonName()
+  showButtonHandler()
   //change the name of the button and will aonly display the starred ideas
 
 }
 
+function showButtonHandler() {
+  if(showStarredIdeasButton.innerText === "Show All Ideas") {
```

Comment:
consider using a class or other attribute to signify whether the button is in a given state, rather than the button text. Will allow for worry free changes to copy/content in the advent that text should change.